### PR TITLE
feat: validate cloud context on root cmd

### PR DESCRIPTION
**Pull request by exu**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: cmd/kubectl-testkube/commands/root.go [hunk 1]
```diff
@@ -62,6 +63,17 @@ var RootCmd = &cobra.Command{
...
+
+		if cmd.Name() == "context" {
+			return
+		}
+
+		cfg, err := config.Load()
+		ui.ExitOnError("loading config", err)
+
+		if err = validator.ValidateCloudContext(cfg); err != nil {
+			common.UiCloudContextValidationError(err)
```
Comment (vsukhin) [inline cmd/kubectl-testkube/commands/root.go:75]: cli should work without os error code for missed config, like PostRun

**Review by vsukhin — COMMENTED** [review]
one important comment

File: cmd/kubectl-testkube/commands/root.go (hunk 1 above)
Comment (exu) [reply cmd/kubectl-testkube/commands/root.go:78]: ok removed it

File: cmd/kubectl-testkube/commands/root.go [hunk 2]
```diff
@@ -62,6 +63,20 @@ var RootCmd = &cobra.Command{
 	Short: "Testkube entrypoint for kubectl plugin",
 	PersistentPreRun: func(cmd *cobra.Command, args []string) {
 		ui.SetVerbose(verbose)
+
+		cfg, err := config.Load()
+		ui.ExitOnError("loading config", err)
```
Comment (vsukhin) [inline cmd/kubectl-testkube/commands/root.go:68]: I would use WarnOnError instead

Comment (vsukhin) [inline cmd/kubectl-testkube/commands/root.go:68]: will it run without kubernetes cluster and ability to load/store testkube config?

**Review by vsukhin — APPROVED** [review]

**Pull request merged**